lib/syscall_shim/Config.uk

```diff
@@ -66,6 +66,7 @@ if LIBSYSCALL_SHIM
 		depends on LIBSYSCALL_SHIM_HANDLER
 			bool "'strace'-like messages for binary system calls"
 			default n
+			select LIBUKDEBUG if !LIBUKCONSOLE
```

Comment:
This is because we call into `ukdebug` if `LIBUKCONSOLE` isn't enabled. See the change in `lib/syscall_shim/uk_syscall_binary.c`. It's funny that we fall back on `ukdebug` for printing if `ukconsole` is missing because `ukdebug` just calls into `ukconsole` internally, so either way if `ukconsole` is disabled, we don't get output. But of course it's not guaranteed that `ukdebug` uses `ukconsole` to print stuff.

Reply:
I think I discussed this with @skuenzer before

Reply:
Okay, let's keep it for now then.